common/app/experiments/Experiments.scala

```diff
@@ -47,3 +48,11 @@ object OldTLSSupportDeprecation extends Experiment(
   // Custom group based on header set in Fastly
   participationGroup = TLSSupport
 )
+
+object AudioPageChange extends Experiment(
+  name = "audio-page-change",
+  description = "Show a different version of the audio page to certain people",
+  owners = Seq(Owner.withGithub("ajwl")),
```

Comment:
I hope you're not off when the test ends 😅 Otherwise you can assign it to the team with `owners = Owner.group(SwitchGroup.Journalism)`